src/components/ProfileBanner.js

```diff
@@ -0,0 +1,48 @@
+import { RiSettings3Fill, RiAddFill } from "react-icons/ri";
+import { useNavigate } from "react-router-dom";
+import axios from "axios";
+
+function ProfileBanner({ image = "", username = "", isMe = false }) {
+  let navitagion = useNavigate();
+  function handleClick() {
+    navitagion("/settings");
+  }
+  async function handleFollow() {
+    const jwt = sessionStorage.getItem("token");
+    await axios({
+      method: "post",
+      url: `${process.env.REACT_APP_API_URL}/api/profiles/${username}/follow`,
+      headers: { Authorization: `Bearer ${jwt}` },
+    });
+  }
+  return (
+    <div className="flex flex-col items-center min-h-[30%] justify-center px-[20%] w-full py-4 bg-tag-color">
+      <img
+        src={`${image}`}
+        className="min-w-20 min-h-20 w-[15%] h-[15%] max-w-2xl max-h-2xl rounded-full"
+        alt="Profile Banner Rounded Avatar"
+      />
+      <p className="my-4 font-bold text-xl">{username}</p>
+      {isMe ? (
+        <button
```

Comment:
Lets make each of these buttons their own component to help with readability and so we can use them in other places potentially.

Reply:
done! created a component named BannerButton.